jakarta.batch.arquillian.extension/src/main/java/ee/jakarta/tck/batch/arquillian/MavenTestDependenciesDeploymentPackager.java

```diff
@@ -165,10 +125,10 @@ private boolean artifactNotToIgnore(MavenResolvedArtifact artifact) {
 
     private void initListOfIgnoredArtifactPrefixes() {
         groupPrefixesToIgnore = new ArrayList<>(Arrays.asList(
-                "org.jboss.shrinkwrap",
-                "arquillian.extension",
-                "org.codehaus.plexus",
-                "org.apache.maven"));
+                "org.jboss.shrinkwrap", // ShrinkWrap - creates a deployment, not needed in the deployment itself
+                "jbatch.arquillian.extension", // This extension - not needed in the deployment itself
```

Comment:
This isn't used like it would seem to be if you saw this line.

Since the filtering is only groupId based it's not going to take effect on the individual artifact.  We'd want to filter out:
    **<artifactId>jakarta.batch.arquillian.extension</artifactId>**

I can't imagine that it's a big deal to leave it in the app unless an Arquillian expert thinks it could get in the way somehow.